src/art/jippy89_3/meta.json

```diff
@@ -0,0 +1,5 @@
+{
+  "art_name": "Digital Ocean Logo",
+  "author_name": "Jip Irfandy",
+  "author_github_url": "https://bit.ly/3TMxuBi"
```

Comment:
Could you update this to use your real Github URL

Reply:
Done @MattCSmith! Thanks for reviewing, have a wonderful day!